test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class OmniauthCallbacksControllerTest < ActionController::TestCase
```

Comment:
How do we know this works without tests? :-1: 

https://github.com/intridea/omniauth/wiki/Integration-Testing